chainer/_version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '7.0.0a1'
```

Comment:
This change seems unrelated. Please revert it.
(ditto for other similar instances)

Reply:
I did this.